projenrc/run-tests-workflow.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { AwsCdkConstructLibrary } from 'projen/lib/awscdk';
+import { JobPermission } from 'projen/lib/github/workflows-model';
+
+export function runTestsWorkflow(project: AwsCdkConstructLibrary) {
+  const runTests = project.github?.addWorkflow('run-tests');
+  if (runTests) {
+    runTests.on({
+      schedule: [
+        { cron: '0 6 * * *' }, // Runs at midnight Mountain Time (UTC-6) every day
```

Comment:
Mountain time, eh?

Reply:
Yeah, it was either east coast or west coast and mountain is kind of in the middle, so it's a little more fair 😁